iotedgedev/modules.py

```diff
@@ -89,8 +89,8 @@ def build_push(self, no_build=False, no_push=False):
             for platform in module_json.platforms:
                 # get the Dockerfile from module.json
                 dockerfile = os.path.abspath(os.path.join(module_dir, module_json.get_dockerfile_by_platform(platform)))
-                container_tag = "" if self.envvars.CONTAINER_TAG == "" else "-" + self.envvars.CONTAINER_TAG
-                tag = "{0}:{1}{2}-{3}".format(module_json.repository, module_json.tag_version, container_tag, platform).lower()
+                #container_tag = "" if self.envvars.CONTAINER_TAG == "" else "-" + self.envvars.CONTAINER_TAG
+                tag = "{0}:{1}{2}-{3}".format(module_json.repository, module_json.tag_version, "", platform).lower()
```

Comment:
I'm probably missing context on this change, but it looks like the new behavior is to always use empty string where we used to potentially use "-{container_tag}". In that case, should we remove the placeholder from the format altogether, and end up with the following?

`tag = "{0}:{1}-{2}".format(module_json.repository, module_json.tag_version, platform).lower()`

Reply:
CONTAINER_TAG is EnvVar that allows you to add additional bits to the tag to make it unique.  That shouldn't be commented out.

Reply:
Accidentally left this commented while testing. Will uncomment